src/RepairManagerAgent/logging.yaml

```diff
@@ -0,0 +1,28 @@
+---
+version: 1
+disable_existing_loggers: False
+formatters:
+  simple:
+    format: '%(asctime)s - %(levelname)s - %(filename)s:%(lineno)d - %(message)s'
+handlers:
+  console:
+    class: logging.StreamHandler
+    level: DEBUG
+    formatter: simple
+    stream: ext://sys.stdout
+  file:
+    class : logging.handlers.RotatingFileHandler
+    formatter: simple
+    filename: repairmanageragent.log
+    # roll over at 10MB
+    maxBytes: 10240000
```

Comment:
Make it 100M, repair manager agent log is quite important.

Reply:
Ok, currently I have 10 backup files at ~10MB each. Should I instead have 1 file of 100MB?

Reply:
Please retain 10 backup count. 1G is not a big deal I think.